### PR TITLE
Committee chair permission

### LINKED ISSUE #3303: Committee chair permissions
### Is your feature request related to a problem? Please describe.
Some committees need certain permissions on the website, but they aren't really necessary for every member of that committee. For example, with #3105, the Thabloidcie should have access to the admin page introduced there. But it would be sufficient, and hence best, if only the committee chair, or perhaps another small subset of the committee has that permission.

### Describe the solution you'd like
I propose extending the `activemembers.backends.MemberGroupBackend` (and `MemberGroupMembership`) to optionally support increased permissions for certain members of a committee. That is:

- We add `MemberGroup.chair_permissions`.
- We add `MemberGroupMembership.has_chair_permissions` (or not and reuse the `chair` field, if we think only allowing the single chair per committee to have increased permissions)
- People with `has_chair_permissions` in their membership can use `Union(permissions, chair_permissions)`

### Motivation
This way we can drop permissions for many people, and increase permissions for a few who need them.

### Describe alternatives you've considered
Leave it as is, give every committee member the same permissions, and rely on the board if someone in a committee needs something done that we find to risky to give everyone permission for.

### PR DESCRIPTION
**Pull request by Noah-marc**
Closes #3303.

### Summary
Added the option to add permissions that only a chair should have to MemberGroups and for  MemberGroupMemberships to have those chair permissions. Also for all existing MemberGroupMemberships that are already marked as chair also automatically have chair permissions 

### How to test
Steps to test the changes you made:
1. Go to a Committee or Society in the admin, select the desired chair-permissions in the respective field and save them
2. Go to the MemberGroupMembership admin and click on a MemberGroupMembership that you want to give chair permissions, mark the checkmark has_chair_permissions and save
3. Make sure that the member connected to that MemberGroupMembership has no other permissions form other MemberGroups
4. Log into that user, go to the admin page and try do the things that you gave permissions to do. It should work then
5. Log into a user that is part of that same MemberGroup and where his Membership does not have those permissions
6. Try to do the same actions as in 4. This time it should fail

### CODE REVIEW

**Review by DeD1rk — COMMENTED** [review]
Looks good as far as I can tell. Is there any chance you could test the migration with a local postgres db (as runpython + field changes has been giving issues on postgres in the past that didn't occur on sqlite3)? 
And maybe it would be nice if someone else also looks at this as I'm biased on the actual permissions backend changes.

**Comment by DeD1rk** [comment]
Oh and yes I see this includes an unrelated commit  (86978fcd087429a3001142cabfcc1576822c2d64) that's causing a conflict. So could you rebase it out? (`git rebase -i origin/master` and `drop` the photo.hidden commit)

File: website/photos/admin.py [hunk 1]
```diff
@@ -101,11 +101,10 @@ class PhotoAdmin(admin.ModelAdmin):
     list_display = (
         "__str__",
         "album",
-        "hidden",
         "num_likes",
     )
     search_fields = ("file",)
-    list_filter = ("album", "hidden")
```
Comment (T8902) [inline website/photos/admin.py:108]: I think removing the hidden picture feature would be better in a different pull request.

**Review by T8902 — CHANGES_REQUESTED** [review]
Removing hidden picture feature should be in a different branch/pull request in my opinion

**Comment by DeD1rk** [comment]
I did the rebase

**Review by DeD1rk — APPROVED** [review]

**Pull request merged**